mongo/description/server.go

```diff
@@ -261,7 +262,7 @@ func NewServer(addr address.Address, response bsoncore.Document) Server {
 				return desc
 			}
 
-			desc.TopologyVersion, err = NewTopologyVersion(doc)
+			desc.TopologyVersion, err = NewTopologyVersion(bsoncore.Document(doc))
```

Comment:
`NewTopologyVersion` should take `bson.Raw`, not `bsoncore.Document` because it now lives in a stable package.

Reply:
done